### ISSUE TITLE
SMAPI Installer fails if .net 6 is installed

### ISSUE THREAD

**Issue by xeropresence** [Pathoschild/SMAPI#815]
Looks like dotnet --info only reports the latest version of even if you install the .net 5 runtime as well
 
```
dotnet --info
.NET SDK (reflecting any global.json):
 Version:   6.0.100
 Commit:    9e8b04bbff

Runtime Environment:
 OS Name:     Windows
 OS Version:  10.0.19043
 OS Platform: Windows
 RID:         win10-x64
 Base Path:   C:\Program Files\dotnet\sdk\6.0.100\

Host (useful for support):
  Version: 6.0.0
  Commit:  4822e3c3aa

.NET SDKs installed:
  6.0.100 [C:\Program Files\dotnet\sdk]

.NET runtimes installed:
  Microsoft.AspNetCore.App 6.0.0 [C:\Program Files\dotnet\shared\Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 5.0.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 6.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by itsanonmelody** [comment]
It looks like you don't have .NET Desktop Runtime 5.0 installed. Did you pick the middle option that says "Run desktop apps" on the download page?
![image](https://user-images.githubusercontent.com/37172966/144358628-971b5b0f-46fe-43ab-9411-4c32a99a604b.png)

**Comment by itsanonmelody** [comment]
I tried installing .NET Desktop Runtime 6.0 and it does not remove the 5.0 version.
```
Host (useful for support):
  Version: 6.0.0
  Commit:  4822e3c3aa

.NET SDKs installed:
  No SDKs were found.

.NET runtimes installed:
  Microsoft.NETCore.App 5.0.12 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.NETCore.App 6.0.0 [C:\Program Files\dotnet\shared\Microsoft.NETCore.App]
  Microsoft.WindowsDesktop.App 5.0.12 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]
  Microsoft.WindowsDesktop.App 6.0.0 [C:\Program Files\dotnet\shared\Microsoft.WindowsDesktop.App]

To install additional .NET runtimes or SDKs:
  https://aka.ms/dotnet-download
```

**Comment by xeropresence** [comment]
Ah, thats probably it. I picked the first option.

I bypassed the check with the installer and smapi still worked fine anyway tho...

**Comment by kuesji** [comment]
issue partially addressed at my pull request ( #816 ).

**Comment by xeropresence** [comment]
If the framework that it's looking for is Microsoft.NETCore.App, then why is the bat file checking for Microsoft.WindowsDesktop.App

**Comment by Pathoschild** [comment]
Fixed in SMAPI 3.13.2, which no longer needs any version of .NET to be installed. Thanks for reporting it!
